docs/pages/visual-test/BannerOverlay.js

```diff
@@ -0,0 +1,38 @@
+// @flow strict
+import { type Node as ReactNode } from 'react';
+import { BannerOverlay, FixedZIndex, Image, Link, Text } from 'gestalt';
+
+export default function Example(): ReactNode {
+  return (
+    <BannerOverlay
```

Comment:
Can you also add other variants like secondary button

Reply:
Done!